arch/arm/boot/dts/overlays/revpi-core-1-1-overlay.dts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Core 1.1
+ */
+
+#include "revpi-core-overlay.dts"
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/{
+	compatible = "brcm,bcm2711", "brcm,bcm2837";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-core-1-1", "brcm,bcm2711";
+			/delete-node/ regulator_pbrst;
+		};
+	};
+
+	fragment@1 {
+		target = <&gpio>;
+		__overlay__ {
+			/delete-node/ eth1_2_reset_pins;
+
+			pileft_pins: pileft_pins {
+				brcm,pins     = <13 21>;
+				brcm,function = <BCM2835_FSEL_GPIO_IN>;
+				brcm,pull     = <BCM2835_PUD_OFF>;
+			};
+
+			piright_pins: piright_pins {
+				brcm,pins     = <12 20>;
+				brcm,function = <BCM2835_FSEL_GPIO_IN>;
+				brcm,pull     = <BCM2835_PUD_OFF>;
+			};
+		};
+	};
+
+	fragment@8 {
+		target = <&spi0>;
+		__overlay__ {
+			pileft: ethernet@0 {
+				pinctrl-names = "default";
+				pinctrl-0 = <&pileft_pins>;
+				interrupt-parent = <&gpio>;
+				#interrupt-cells = <2>;
+				interrupts = <13 IRQ_TYPE_LEVEL_LOW>;
+				reset-gpios = <&gpio 21 GPIO_ACTIVE_HIGH>;
+				vdd-supply = <>;
+			};
+
+			piright: ethernet@1 {
+				pinctrl-names = "default";
+				pinctrl-0 = <&piright_pins>;
+				interrupt-parent = <&gpio>;
+				#interrupt-cells = <2>;
+				interrupts = <12 IRQ_TYPE_LEVEL_LOW>;
+				reset-gpios = <&gpio 20 GPIO_ACTIVE_HIGH>;
+				vdd-supply = <>;
+			};
+		};
+	};
+};
```

Comment:
eth0_mac_hi and eth0_mac_left are not used any more. We have an eeprom attached to the USB phys.

Reply:
I assume you mean the eth0_mac_* is not needed any more.   

And how about pileft_mac_* and piright_mac_* ? Can the be obtained from hat-eeprom or some where?  But anyway, it seems they are never gotten used, am I right?

Reply:
I deleted the eth0_mac_*